src/Compatibility/Microsoft.DotNet.ApiCompatibility/Rules/EnumValuesMustMatch.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.DotNet.ApiCompatibility.Abstractions;
+
+namespace Microsoft.DotNet.ApiCompatibility.Rules
+{
+    public class EnumValuesMustMatch : Rule
+    {
+        public override void Initialize(RuleRunnerContext context)
+        {
+            context.RegisterOnTypeSymbolAction(RunOnTypeSymbol);
+        }
+
+        private static bool isEnum(ITypeSymbol type)
+        {
+            return type != null && type.IsType && type.TypeKind == TypeKind.Enum;
+        }
+        private void RunOnTypeSymbol(ITypeSymbol left, ITypeSymbol right, string leftName, string rightName, IList<CompatDifference> differences)
+        {
+            if (!isEnum(left) || !isEnum(right))
```

Comment:
Why do you have to check if both are Enum? I would expect that it would never be the case where the Mapper would give you an Enum on the left but not an enum on the right. Perhaps what you want to check here is only if one of them is an enum, and that should be enough.

Reply:
I believe I saw this method being run where `left` was `null`. I'll double-check to make sure that `right` was an `Enum` in that circumstance.